docs/configuration/modules/remote-service.md

```diff
@@ -31,6 +31,28 @@ to connect to the remote server. The default value of this property is `true`.
 The `port` property is an optional property that defines the connection port. By default, if SSL/TLS protocol
 is used, the default value of this property is `443`. Otherwise, it is `80`.
 
+## autoreconnect-enabled
+
+The `autoreconnect-enabled` property is an optional property that defines if auto reconnection to the server should be used when to disconnect. The default value of this property is `false`.
+
+## reconnection-initial-interval
+
+The `reconnection-initial-interval` property is an optional property that defines the reconnection initial interval. The default value of this property is `5`in seconds.
```

Comment:
Be more explicit: "[...] defines the initial interval (in seconds) before the first reconnection. Its default values is `5`."

Reply:
It's done